### PR TITLE
Clarified means of "store records" setting in Umbraco Forms when it comes to file uploads.

### PR DESCRIPTION
**Pull request by AndyButland**
## Description

Clarified means of "store records" setting when it comes to file uploads.

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [ ] New content
* [ ] Updates related to a new version
* [ ] Other

## Product & version (if relevant)

Forms 10+

## Deadline (if relevant)

Can be reviewed and published at any time.

### CODE REVIEW

File: 10/umbraco-forms/editor/creating-a-form/form-settings.md [hunk 1]
```diff
@@ -16,7 +16,9 @@ The following options are available in Forms Settings:
 
 ### Store Records
 
-By default, all submitted records are saved in the database. This option allows you to view and export the saved records from the queries overview. If you do not want to store data (due to policies in your organisation), you can uncheck the box.
+By default, all submitted records are saved in the database. This option allows you to view and export the saved records from the queries overview. If you do not want to store data (due to policies in your organization), you can uncheck the box.
+
+Disabling this option will prevent database records from being stored, but any file uploads made as part of the form submission are retained. If you do not want these stored, any method used to process or copy them to a different location should also remove the file.
```
Comment (eshanrnh) [inline 10/umbraco-forms/editor/creating-a-form/form-settings.md:21]:
```suggestion
Disabling this option will prevent database records from being stored, but any file uploads made as part of the form submission will still be retained. If you do not want the files to be stored, ensure that any process or method used to process, move, or copy them to a different location also removes the file.
```

File: 12/umbraco-forms/editor/creating-a-form/form-settings.md [hunk 2]
```diff
@@ -16,7 +16,9 @@ The following options are available in Forms Settings:
 
 ### Store Records
 
-By default, all submitted records are saved in the database. This option allows you to view and export the saved records from the queries overview. If you do not want to store data (due to policies in your organisation), you can uncheck the box.
+By default, all submitted records are saved in the database. This option allows you to view and export the saved records from the queries overview. If you do not want to store data (due to policies in your organization), you can uncheck the box.
+
+Disabling this option will prevent database records from being stored, but any file uploads made as part of the form submission are retained. If you do not want these stored, any method used to process or copy them to a different location should also remove the file.
```
Comment (eshanrnh) [inline 12/umbraco-forms/editor/creating-a-form/form-settings.md:21]:
```suggestion
Disabling this option will prevent database records from being stored, but any file uploads made as part of the form submission will still be retained. If you do not want the files to be stored, ensure that any process or method used to process, move, or copy them to a different location also removes the file.
```

File: 13/umbraco-forms/editor/creating-a-form/form-settings.md [hunk 3]
```diff
@@ -16,7 +16,9 @@ The following options are available in Forms Settings:
 
 ### Store Records
 
-By default, all submitted records are saved in the database. This option allows you to view and export the saved records from the queries overview. If you do not want to store data (due to policies in your organisation), you can uncheck the box.
+By default, all submitted records are saved in the database. This option allows you to view and export the saved records from the queries overview. If you do not want to store data (due to policies in your organization), you can uncheck the box.
+
+Disabling this option will prevent database records from being stored, but any file uploads made as part of the form submission are retained. If you do not want these stored, any method used to process or copy them to a different location should also remove the file.
```
Comment (eshanrnh) [inline 13/umbraco-forms/editor/creating-a-form/form-settings.md:21]:
```suggestion
Disabling this option will prevent database records from being stored, but any file uploads made as part of the form submission will still be retained. If you do not want the files to be stored, ensure that any process or method used to process, move, or copy them to a different location also removes the file.
```

File: 14/umbraco-forms/editor/creating-a-form/form-settings.md [hunk 4]
```diff
@@ -16,7 +16,9 @@ The following options are available in Forms Settings:
 
 ### Store Records
 
-By default, all submitted records are saved in the database. This option allows you to view and export the saved records from the queries overview. If you do not want to store data (due to policies in your organisation), you can uncheck the box.
+By default, all submitted records are saved in the database. This option allows you to view and export the saved records from the queries overview. If you do not want to store data (due to policies in your organization), you can uncheck the box.
+
+Disabling this option will prevent database records from being stored, but any file uploads made as part of the form submission are retained. If you do not want these stored, any method used to process or copy them to a different location should also remove the file.
```
Comment (eshanrnh) [inline 14/umbraco-forms/editor/creating-a-form/form-settings.md:21]:
```suggestion
Disabling this option will prevent database records from being stored, but any file uploads made as part of the form submission will still be retained. If you do not want the files to be stored, ensure that any process or method used to process, move, or copy them to a different location also removes the file.
```

**Review by eshanrnh — COMMENTED** [review]
Thanks for the PR, @AndyButland 🙌 I have added a suggestion to clarify the statement. Do you think it makes sense?

**Comment by AndyButland** [comment]
Yes, fine with me, have accepted - thanks.

**Review by eshanrnh — APPROVED** [review]

**Pull request merged**